src/platforms/browser/webgl/WebGLContext2D.js

```diff
@@ -145,7 +145,7 @@ var GLManager = Class(function() {
     this._canvas.getWebGLContext = this._canvas.getContext.bind(this._canvas, 'webgl', {
       alpha: true,
       premultipliedAlpha: true,
-      preserveDrawingBuffer: true
+      preserveDrawingBuffer: CONFIG.preserveDrawingBuffer
```

Comment:
any default value needed?

Reply:
good question. what happens when it is undefined.

Reply:
updated with default values